web/redirect.html

```diff
@@ -4,55 +4,183 @@
     <meta charset="utf-8"/>
     <title>LBRY Redirect</title>
 
-    <link rel="stylesheet" href="lbry-color.css"/>
     <link rel="stylesheet" href="reset.css"/>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/lbryio/components@master/dist/index.css"/>
 
     <style>
       body {
-        background-color: var(--lbry-cyan-5);
         background-image: linear-gradient(to bottom right, var(--lbry-teal-5), var(--lbry-cyan-5) 100%);
+      }
+
+      button {
         color: var(--lbry-white);
       }
 
-      section {
+      checkbox-element {
+        margin: 0 auto var(--spacing-l);
+      }
+
+      .container {
+        align-items: center;
+        background-color: var(--lbry-white);
+        border-radius: var(--spacing-xxs);
+        display: flex;
+        flex-direction: column;
+        font-size: 1.25rem;
+        max-width: 500px;
+        padding: var(--spacing-l);
         text-align: center;
       }
 
-      p:not(:last-of-type) {
+      .container__header {
         margin-bottom: 2rem;
       }
 
-      a {
-        background-color: var(--lbry-white);
-        border-radius: 0.2rem;
-        color: var(--lbry-cyan-5);
-        font-size: 1.2rem;
-        padding: 0.2rem 0.5rem;
-        text-decoration: none;
+      .container__header small {
+        color: var(--lbry-gray-4);
+        display: block;
+        padding-top: var(--spacing-xxs);
+      }
+
+      .control-container {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        justify-content: space-around;
+        padding-top: var(--spacing-s);
+        padding-bottom: var(--spacing-s);
+      }
+
+      .control-container *:first-child {
+        margin-right: var(--spacing-m);
       }
     </style>
+
+    <script src="analytics.js"></script>
   </head>
 
   <body>
-    <section>
-      <h1>Buckle up, gang!</h1>
-      <p>Redirect in progress...</p>
-      <p>If you do not have the LBRY app, you can <a href="https://lbry.io/get" rel="noopener noreferrer" target="_blank">download it here</a>.</p>
+    <section class="container">
+      <header class="container__header">
+        <button class="button--primary" id="stopButton">
```

Comment:
I think making this a button is a mistake. This results in 4 primary styled buttons inside of the same component. There should probably only be 1 or 2.